src/components/RegistrationFlow/Picture/index.js

```diff
@@ -50,18 +75,83 @@ const Picture = props => {
         </div>
 
         <div className="add-user-picture-content-container">
-          <Avatar
-            width={115}
-            height={115}
-            label="&nbsp;"
-            labelStyle={customLabelStyle}
-            borderStyle={customBorderStyle}
-            onBeforeFileLoad={onBeforeFileLoad}
-            onCrop={onChange}
-            className="add-user-picture-picker"
-          />
+          {showFileLoader ? (
+            <ImageUploader
+              onChange={setImageFile}
+              setShowFileLoader={setShowFileLoader}
+            />
+          ) : (
+            <div className="image-preview-container">
+              <ButtonContainer
+                className="icon-btn close-preview-button"
+                onClick={closePreview}
+              >
+                <i className="fas fa-times" />
+              </ButtonContainer>
+              <AvatarEditor
+                ref={editorRef}
+                image={imageFile}
+                width={200}
+                height={200}
+                border={50}
+                borderRadius={100}
+                color={[255, 255, 255, 0.8]} // RGBA
+                scale={parseFloat(scale)}
+                rotate={parseFloat(rotation)}
+                position={position}
+                onPositionChange={handlePositionChange}
+                className="add-user-picture-picker"
+                onImageReady={savePreview}
+                onImageChange={savePreview}
+              />
+
+              <div className="controls-button-container">
+                <ButtonContainer
+                  className="icon-btn rotate-btn"
+                  onClick={handleRotate}
+                >
+                  <i className="fas fa-retweet" />
+                </ButtonContainer>
+                <br />
+                <div className="zoom-controls-container">
+                  <i className="fas fa-minus zoom-control-sign" />
+                  <Slider
```

Comment:
Good idea to add the slider! I was trying to use the + and - signs as buttons, and I think that would be easier with mobile devices for resizing the image. Maybe we can add that if there is somehow still time.

Reply:
We were actually thinking about that too. Maybe I'll do it now, it shouldn't be too much work.

Reply:
I added these in now, making the zoom step a little bigger to avoid having to click so much.

Reply:
Looks good 👍